### DISCUSSION TITLE
Old header style

### DISCUSSION THREAD

**Discussion by Daantjerrr** [chrisgrieser/shimmering-focus#210] [Q&A] [answered]
Hi! I just updated to the new insider build, and also got the new version of Shimmering Focus. It looks great! But there's one thing I like a little less (though it might be a matter of getting used to it): headers no longer having their own little 'boxes' (not sure how to describe this in another way). I really liked that feature, as it really helped me to visually divide longer notes (like my daily note and project notes). 
Would it be possible to add an option for the old-style headers in style settings, or as a CSS snippet? 

(and: I would like to stress how much I appreciate the work you put and the theme and all work you must do to port it to 0.16, so no pressure!)

**Comment by chrisgrieser** [comment, marked as answer]
Hi,

yeah I haven't had the time to recreate all features from the old shimmering focus yet. 

Please open a GitHub issue, and I'll get to the most upvoted features as soon as I have the time! 🙂

**Comment by Daantjerrr** [comment]
Thanks for your swift reply! I'll do that (and wait patiently).
